web/hosts.go

```diff
@@ -123,9 +123,16 @@ func NewHostHandler(hostsService services.HostsService, subsService services.Sub
 			return
 		}
 
+		jobsState, err := hostsService.GetExportersState(host.Name)
```

Comment:
Let's go by parts hehe:
- I chose this name because we are actually showing the exporters state. In fact, we want (in my opinion) to make it explicit, that we are referring to the prometheus exporters state. If something is in red in the Web, the user should check if this particular exporter is Ok or not. I'm afraid that we are not in a level to hide all the technical details. The user knows, and must know, that they are using prometheus exporters as part of the solution, because most probably, they will need to do some operations in there. And this operations belong to one particular host. 
That's why I included there. Now, you can very well say: "No, we don't want the users knowing anything about prometheus and exporters. that's just a tech detail". This will depend on the level of customization we want to give to the user. If they should be able to create their own metrics, alerts, graphs, etc, they will need to know what they need to know.
- About the `whole page fails` thing. I'm not sure what do you mean here. In the long term, more we use prometheus metrics, more we depend on it, so not having prometheus running would totally affect the page. In any case, not having some exporter wouldn't make the page fail. This is actually what this PR aims to show. Not having exporters would only mean that we would have this state in red. If prometheus server is not available, yes, the whole app would fail, in the same way if we don't have access to the postgres DB.

Reply:
comparing prometheus to postgres as a hard dependency is a good argument, but let's have a chat about it to see if we're missing something.